src/messaging/ExchangeTransport.h

```diff
@@ -0,0 +1,86 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      This file defines ExchangeTransport class. The object of this
```

Comment:
The name of ExchangeTransport is really confusing, I always confuse it with the low level Transports, could we come up with a better name, such as ExchangeBinding or ExchangeConnection?

Reply:
I agree. I am also not very happy with the name. How about `ExchangeMessageDispatch`?